helm/flowforge/templates/broker.yaml

```diff
@@ -153,6 +156,12 @@ spec:
                 name: flowforge-broker
                 port:
                   number: 1884
+  {{- if .Values.ingress.certManagerIssuer }}
+  tls:
+  - hosts:
+    - mqtt.{{ .Values.forge.domain }}
+    secretName: broker-tls
```

Comment:
I suggest renaming secret name to the value of domain name (`mqtt.{{.values.forge.domain}}` in this case).

Reply:
Make sense, then it's more obvious what it is